src/Controller/Api/NodeControllerValidation.ts

```diff
@@ -19,6 +20,7 @@ export const CreateNodeValidationSchema = Joi.object({
         url: Joi.string().required(),
         address: Joi.string().required(),
     }),
+    notifications: Joi.bool().required()
```

Comment:
We can leave this supported for the daemon in the future but don't put `required` for now as it won't work until we support it in the daemon (which we might do later at some point)